### PR TITLE
Eda

### PR DESCRIPTION
**Pull request by FiiiishX**
Final Version

### CODE REVIEW

File: Group_16.Rmd [hunk 1]
```diff
@@ -40,160 +48,761 @@ data<-load("2000data.RData")
...
+WestEU: 'Austria', 'Germany', 'England'	
 
-# Method
-## Area
+EastEU: 'Greece', 'Romania', 'Croatia', 'Israel', 'Bulgaria', 'Georgia', 'Slovenia', 'Turkey', 'Uruguay', 'Czech Republic', 'Hungary'
 
-```{r}
-NorthAmerica=filter(data_without_NAs,country=='US'|country=='Canada')
- NorthAmerica=filter(data_without_NAs,country=='US'|country=='Canada')
+```{r group country by area}
```
Comment (HEIYO123) [inline Group_16.Rmd:288]: Good

File: Group_16.Rmd [hunk 2]
```diff
@@ -40,160 +48,761 @@ data<-load("2000data.RData")
...
+  drop_na(processed_winery) %>% 
+  glm(above90 ~ processed_winery  , data = ., family = binomial(link = "logit"))
+
+model_winery %>% 
+  summary()
+```
+
+Larger wineries are more likely to score above 90 than smaller wineries.
+
+## Full model
```
Comment (HEIYO123) [inline Group_16.Rmd:544]: Full model is good

**Review by HEIYO123 — APPROVED** [review]
This merge is reasonable.

**Review by MingranJia — APPROVED** [review]
Approve

**Pull request merged**